plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -195,6 +198,9 @@ test.describe( 'Add New Variable Product Page', () => {
 		await page.click( 'a[href="#variable_product_options"]' );
 		await page.waitForLoadState( 'networkidle' );
 		await page.selectOption( '#field_to_edit', 'delete_all' );
+		await page.evaluate( () => {
+			window.confirm = () => true;
+		} );
```

Comment:
Why is this needed? We should comment this, since it isn't clear.

This is mocking the `confirm` browser dialog, right? Why is this needed?

Would it make more sense to do...

```
page.on('dialog', dialog => dialog.accept());
```

From: https://playwright.dev/docs/dialogs

Reply:
After rebasing my PR, there were introduced a few changes that broke that part and this is why I mocked the `confirm`. I realized that the rebase left a duplicated `on dialog` and removing one of them fixed the problem.